roles/matrix-backup-borg/templates/config.yaml.j2

```diff
@@ -0,0 +1,37 @@
+#jinja2: lstrip_blocks: "True", trim_blocks: "True"
+
+location:
+  source_directories:
+  - {{ matrix_backup_borg_source }}
+  repositories:
+  - {{ matrix_backup_borg_repository }}
```

Comment:
Perhaps `repositories: {{ matrix_backup_borg_repositories|to_json }}` is more flexible here?

You can define:

```yaml
matrix_backup_borg_repositories: ["{{ matrix_backup_borg_repository }}"]
```

Or just get rid of `matrix_backup_borg_repository` and ask people to configure the `matrix_backup_borg_repositories` list?